Trackovid19-web/src/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Covidografia",
+    "short_name": "Covidografia",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone",
+    "scope": "./",
+    "start_url": "./",
+    "icons": [
+        {
+            "src": "assets/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "assets/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ]
+}
```

Comment:
There are a couple more properties we can add to this manifest file.

Reply:
Yes, I know. But can you give more detail, which ones are useful to have?

Reply:
For example, adding `"purpose": "maskable"` to each icon object will be consistent across OS versions.

Reply:
I revisited the manifest file and I added some more properties, including `"purpose": "maskable"`. Can you check if this is ready now?

Reply:
Yes, looks good 👌.